docs/platform/concepts/service_backups.rst

```diff
@@ -60,6 +60,9 @@ Aiven for Apache Kafka®
 '''''''''''''''''''''''
 
 Aiven for Apache Kafka is usually used as a transport tool for data rather than a permanent store. Due to the way it stores data, traditional backup strategies are not feasible. As a result, Aiven does not perform backups for managed Apache Kafka services, and data durability is determined by data replication across the cluster.
+But, we include :doc:`configuration backups </docs/products/kafka/concepts/configuration-backup>` that automatically back up key Apache Kafka® service configurations for you without any additional costs. 
+
+Suppose the Apache Kafka® service is powered off/on or an incident causes the Apache Kafka cluster to stop functioning. In that case, the configuration backup enables the restoration of your Apache Kafka® service to its previous state.
```

Comment:
```suggestion
However, automatic backups for essential Apache Kafka® service configurations are offered through :doc:`configuration backups </docs/products/kafka/concepts/configuration-backup>`, and they come at no extra cost.

If the Apache Kafka® service is powered off/on or if any incidents lead to the cluster's failure, the configuration backup facilitates restoring your Apache Kafka® service to its previous state.
```

Reply:
Thanks for the suggestions. I have resolved the same.